extension/chrome/settings/modules/backup-manual-module.ts

```diff
@@ -53,17 +54,78 @@ export class BackupManualActionModule extends ViewModule<BackupView> {
     }
   }
 
+  private addKeyToBackup = (prvKeyIdentity: PrvKeyIdentity) => {
+    this.view.prvKeysToManuallyBackup.push(prvKeyIdentity);
+  }
+
+  private removeKeyToBackup = (fingerprints: string[]) => {
+    this.view.prvKeysToManuallyBackup.splice(this.view.prvKeysToManuallyBackup.findIndex(prvIdentity => prvIdentity.fingerprints === fingerprints), 1);
+  }
+
+  private preparePrvKeysBackupSelection = async () => {
+    const primaryKeys = await KeyStore.getAllWithOptionalPassPhrase(this.view.acctEmail);
+    if (primaryKeys.length > 1) {
+      await this.renderPrvKeysBackupSelection(primaryKeys);
+    } else {
+      this.addKeyToBackup({ 'email': String(primaryKeys[0].emails), 'fingerprints': primaryKeys[0].fingerprints });
+    }
+  }
+
+  private renderPrvKeysBackupSelection = async (primaryKeys: ExtendedKeyInfo[]) => {
+    for (const primaryKi of primaryKeys) {
+      const email = Xss.escape(String(primaryKi.emails));
```

Comment:
This is how it looks now.
![image](https://user-images.githubusercontent.com/23335944/135439316-6ad7c7f6-41ec-49cb-a557-1ebdce83a0a1.png)
@martgil Do you really want to display all emails the key may have? I would use `primaryKi.emails.join()` or `primaryKi.emails.toString()` for extra build-time type check -- this is why we're using Typescript after all. Have you tested how it would look visually with more than one email?
I guess `email[0]` is enough, @tomholub ?

Reply:
> Have you tested how it would look visually with more than one email?

Yes, sir! I make the container available for scrolling when it reaches the maximum height of 300px.